episodes/06-working-with-columns.md

```diff
@@ -34,7 +34,7 @@ You can sort data in OpenRefine by clicking on the drop-down menu for the column
 
 Once applied, locate the new "Sort" button at the top of the grid.
 
-![](fig/sort-menu-highlight.png){alt='Addition of Sort menu to OpenRefine grid after first sort command'}
+![New Sort menu appears above grid after first sort command](fig/sort-menu-highlight.png){alt="Note the addition of a menu called Sort appears after your first sort command. It follows the number of rows displayed."}
```

Comment:
This doesn't read quite correctly to me. Should it be:

Note the addition of a menu called Sort **which** appears after your first sort command. It follows the number of rows displayed.

?